shiny/_docstring.py

```diff
@@ -150,6 +193,86 @@ def _(func: F) -> F:
     return _
 
 
+def is_express_app(app_path: str) -> bool:
+    # We can't use .shiny.express._is_express.is_express_app() here because that would
+    # create a circular import.
```

Comment:
Import here, avoid circular import

Reply:
Oof, this still gave me a circular import error

```
  File "/Users/garrick/work/posit-dev/py-shiny/shiny/ui/_accordion.py", line 11, in <module>
    from ..session import require_active_session
ImportError: cannot import name 'require_active_session' from partially initialized module 'shiny.session' (most likely due to a circular import) (/Users/garrick/work/posit-dev/py-shiny/shiny/session/__init__.py)
```

I'm going to leave this alone for now